engines/savestate.cpp

```diff
@@ -95,7 +95,12 @@ bool SaveStateDescriptor::isAutosave() const {
 
 bool SaveStateDescriptor::hasAutosaveName() const
 {
-	return _description.contains(_("Autosave"));
+	Common::U32String autosave = _("Autosave");
```

Comment:
const

Reply:
Seeing that code now I am wondering if there might be a better way to test if a savegame is an autosave. Since the description string can be translated this means that after changing the GUI language in the ScummVM option, existing autosaves will no longer be recognised as such...

Reply:
maybe something with SeekableReadStream? or a secondary meta file?

I just wanted to fix the issue my friend was having where he was constantly getting spammed with the autosave warning dialog

Reply:
> Seeing that code now I am wondering if there might be a better way to test if a savegame is an autosave. Since the description string can be translated this means that after changing the GUI language in the ScummVM option, existing autosaves will no longer be recognised as such...

I've noticed that too, and I even commented on this issue in #3261, but I wanted to preserve the existing behavior too.